common/djangoapps/student/admin.py

```diff
@@ -22,7 +22,7 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 
-from edx_toggles.toggles import LegacyWaffleSwitch
+from edx_toggles.toggles import WaffleSwitch
 from openedx.core.lib.courses import clean_course_id
 from common.djangoapps.student import STUDENT_WAFFLE_NAMESPACE
```

Comment:
I think you can kill the import, right?  Seems like `STUDENT_WAFFLE_NAMESPACE` can be removed completely, because I think this was the only usage.

Reply:
I removed the import and also STUDENT_WAFFLE_NAMESPACE from the __init__.py file